CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+First off, thank you for considering contributing to Suitest JS API.
+
+When contributing to this repository, please first discuss the change you wish to make via GitHub issue
+with the owners of the repository.
+
+Please note we have a [code of conduct][]. Follow it in all your interactions with the project.
+
+## 1. How do I start contributing?
+
+If you've noticed a bug, have a question or a suggestion, [search the issue tracker][] to see
+if someone has already create a ticket. If not, go ahead and [make one][new issue].
+
+## 2. Fork & create a branch
+
+If this is something you think you can fix, then [fork Suitest JS API][] and
+create a branch with a descriptive name.
+
+We use `hotfix/[issue-id]-[issue-title]` convention for small changes. For example:
```

Comment:
i don't think we need to force this convention onto the contributers, especially since they don't have access to THE issue tracker.

Reply:
By issue tracker I mean the GitHub one's. The only part that makes sense to drop is "hotfix/", but that's a not big deal anyway, right?